<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialogCourseInfo</class>
 <widget class="QDialog" name="QDialogCourseInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EnterRoom</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMain" native="true">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>265</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>265</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widgetMain{
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>26</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>26</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>11</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="minimumSize">
             <size>
              <width>11</width>
              <height>11</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>11</width>
              <height>11</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-image: url(:/Course/course/close-normal.png);
}

QPushButton:pressed {
	border-image: url(:/Course/course/close-press.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="title">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">#title{
	image: url(:/Course/course/logo-portal.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_input" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget_input{
	border: 1px solid #dddddd;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QLineEdit" name="tb_court_name">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>39</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	font: 9pt &quot;Microsoft YaHei&quot;;
    color: #888888;
	border: 1px solid #ffffff;
	border-bottom-color: #f2f2f2;
	padding-left: 10px;
}</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
             <property name="placeholderText">
              <string>请输入课程名称</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tb_nick_name">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>39</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	font: 9pt &quot;Microsoft YaHei&quot;;
    color: #bbbbbb;
	border: 1px solid #ffffff;
	padding-left: 10px;
}</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
             <property name="placeholderText">
              <string>请输入你的昵称</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>24</number>
        </property>
        <property name="bottomMargin">
         <number>24</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_operate">
          <property name="minimumSize">
           <size>
            <width>76</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: none;
	background:  #006eff;
	color: #ffffff;
	font: 10pt &quot;Microsoft YaHei&quot;;
}

QPushButton:hover {
	background:  #0068f2;
}

QPushButton:pressed {
	background:  #0062e5;
}

QPushButton:disable {
	border: 1px solid #d5d5d5;
	color: #bbbbbb;
	background:  #f2f2f2;
}</string>
          </property>
          <property name="text">
           <string>创建</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="minimumSize">
           <size>
            <width>76</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>76</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	border: 1px solid #dddddd;
	background:  #ffffff;
	color: #000000;
	font: 10pt &quot;Microsoft YaHei&quot;;
}

QPushButton:hover {
	border: 1px solid #d2d2d2;
	background:  #f2f2f2;
}

QPushButton:pressed {
	border: 1px solid #d2d2d2;
	background:  #e5e5e5;
}

QPushButton:disable {
	border: 1px solid #d5d5d5;
	color: #bbbbbb;
	background:  #f2f2f2;
}</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tb_court_name</tabstop>
  <tabstop>tb_nick_name</tabstop>
  <tabstop>btn_operate</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>btn_close</tabstop>
 </tabstops>
 <resources>
  <include location="../res/Course.qrc"/>
 </resources>
 <connections/>
</ui>
